docs/ballerina-by-example/examples/filter-connector/filter-connector.bal

```diff
@@ -28,19 +32,22 @@ connector CacheConnector<StockQuoteConnector stockC> (string j) {
 }
 
 function main (string[] args) {
-    // Create the 'StockQuoteConnector' with 'CacheConnector' as the filter connector
+    // Create the 'StockQuoteConnector' that acts as the base connector and decorate it using the 'CacheConnector'
+    that acts as the filter connector.
     StockQuoteConnector stockQC = create StockQuoteConnector(5)
                                   with CacheConnector("Bob");
 
-    // Invoke the action of the 'StockQuoteConnector' with a cached key 'WSO2'.
+    // Invoke the action of the 'StockQuoteConnector' with the 'WSO2' cached key that was defined in 'cachedKeys' of the
+     filter connector.
```

Comment:
Comment missing.. Also, didn't understand above statement. What do we mean by following:

'WSO2' cached key that was defined in 'cachedKeys' of the filter connector.

Reply:
Wanted to say that if the given value is in the cache of the filter connector then that price of that value is printed. Reworded is as follows:

"Invoke the action of the 'StockQuoteConnector' by passing 'WSO2' as the ID. Since WSO2 is stored in the 'cachedKeys' map of the filter connector the respective price value is printed."